example/index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+		<link rel="stylesheet" href="style.css" />
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap"
+			rel="stylesheet"
+		/>
+		<title>Document</title>
+	</head>
+
+	<body>
+		<div class="container">
+			<div class="title">
+				<img
+					class="profile-photo"
+					src="./assets/Default_pfp.svg.png"
+					alt="Profile photo"
+				/>
+				<span class="person-name">Maksim Jaroslacevas</span>
+			</div>
+			<div class="last-time-seen">
+				<div class="icon">

```

Comment:
There is no need for separate `div` wrapper, as you can pass `class` directly to `svg` or `img`. Same with all icons.